.github/workflows/gh-pages.yaml

```diff
@@ -3,36 +3,58 @@ name: CI/CD
 on: push
 
 env:
-  # Creates and uploads sourcemaps to Rollbar, and save the built extension as an artifact
   PUBLIC_RELEASE: ${{ github.ref == 'refs/heads/main' }}
 
 jobs:
-  build-stage:
+  build:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v2
       - uses: actions/setup-node@v2
         with:
           node-version: '14'
+
+      #########################
+      # Prepare public folder #
+      #########################
+
+      - name: Create public distribution
+        run: |
+          mkdir -p public
+          mv example public
+        # Duplicate the line above to add more folders
+
+
+      #########
+      # Build # Each build should be self-contained
```

Comment:
To avoid having to keep track of changes across the whole workflow, it's best if people can focus on a single job at a time.

This means that the "Build react-example" job installs, build _and_ moves the built files to the public folder, in one go.